### DISCUSSION TITLE
No list in admin view

### DISCUSSION THREAD

**Discussion by EvDoorne** [joomengine/Joomla-Component-Builder#696] [Q&A]
Newbie question here: I made a simple component with 1 table with 2 fields, 1 admin view using these fields and no site view or custom view. I looked at all settings from the demo component and made sure my component and admin view have identical settings. Compilation and installation goes flawless.

When I open the admin view and add a record, the admin view does not show a list of records. I checked the database table and the records are added alright, there's content in the table attached to the admin view. But I can't get the amdin view to show the list of records.

I'm a bit out of ideas. Has anybody experienced this before and point me to a possible cause and/or solution?

Thanks in advance, Erik

**Comment by multizone-uk** [comment]
I had this, and am very much a newbie too. I had to change an entry for the field that was set to 'default' and then recompile.

**Comment by EvDoorne** [comment]
Thanks for your input @multizone-uk . I checked the field settings, made some adjustments and that did the trick. Solved.
